src/functions_framework/__init__.py

```diff
@@ -106,6 +137,26 @@ def _run_cloud_event(function, request):
     function(event)
 
 
+def _typed_event_func_wrapper(function, request, inputType: Type):
+    def view_func(path):
+        try:
+            data = request.get_json()
+            input = inputType.from_dict(data)
```

Comment:
Can we add tests that trigger these two lines?

1. malformed json body in POST
2. from_dict just throws an exception or something